kolibri/deployment/default/settings/base.py

```diff
@@ -96,11 +96,17 @@
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': os.path.join(KOLIBRI_HOME, 'db.sqlite3'),
+    },
+    'ormq': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(KOLIBRI_HOME, 'ormq.sqlite3'),
     }
 }
 
 # Enable dynamic routing for content databases
-DATABASE_ROUTERS = ['kolibri.content.content_db_router.ContentDBRouter']
+DATABASE_ROUTERS = ['django_q.router.ORMBrokerRouter',
+                    # note: the content db router seems to override any other routers you put in here. Make sure it's the last.
+                    'kolibri.content.content_db_router.ContentDBRouter']
```

Comment:
Interesting -- we do return `None` for the main routing methods:
```
        if not issubclass(model, ContentDatabaseModel):
            return None
```

But it may be something in the `allow_migrate` logic, which does a binary split between default and non-default DBs,  but should probably be returning an unopinionated `None` in some cases.